libs/application/templates/reference-template/src/forms/messages.ts

```diff
@@ -13,7 +13,8 @@ export const m = defineMessages({
   },
   introIntroduction: {
     id: 'example.application:intro.introduction',
-    defaultMessage: 'Halló {name}',
+    defaultMessage:
```

Comment:
this should not be part of the PR

Reply:
No, I forgot I branched out of another branch, not master. That should be merged, will rebase